### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        inputs:
+        - new_version: 0.9.0
+    - script:
+        title: Output test
+        inputs:
+        - content: |-
+            echo '$BITRISE_CHANGELOG='
+            echo $BITRISE_CHANGELOG
+            if [[ -z $BITRISE_CHANGELOG ]] ; then
+              echo "empty $BITRISE_CHANGELOG"
```
Comment (trapacska) [inline bitrise.yml]: it is not `"`

File: bitrise.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        title: Audit step.yml
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            stepman audit --step-yml ./step.yml
+
+  release:
+    envs:
+    - BITRISE_STEP_GIT_CLONE_URL: https://github.com/godrei/generate-changelog.git
```
Comment (trapacska) [inline bitrise.yml]: change urls

File: bitrise.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    - BITRISE_STEP_ID: generate-changelog
+    - BITRISE_STEP_VERSION: $BITRISE_GIT_TAG
+    - BITRISE_GITHUB_API_TOKEN: $BITRISE_GITHUB_API_TOKEN
+    - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+    before_run:
+    - test
+    steps:
+    - git::https://github.com/godrei/steps-generate-changelog@master:
```
Comment (trapacska) [inline bitrise.yml]: wire in final steps

File: changelog.go [hunk 4]
```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"html/template"
+	"sort"
+	"time"
+
+	"github.com/godrei/steps-generate-changelog/git"
```
Comment (trapacska) [inline changelog.go]: fix path

File: bitrise.yml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    steps:
+    - script:
+        title: Audit step.yml
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            stepman audit --step-yml ./step.yml
+
+  release:
```
Comment (godrei) [inline bitrise.yml]: i think for now we should use the 'old' release wf.
the listed steps are not yet released (also the step ids are not correct): `steps-generate-changelog`, `steps-github-release`

**Pull request merged**